cpp/open3d/visualization/visualizer/O3DVisualizer.cpp

```diff
@@ -1762,8 +1762,10 @@ O3DVisualizer::O3DVisualizer(const std::string &title, int width, int height)
 #endif  // __APPLE__
     auto file_menu = std::make_shared<Menu>();
```

Comment:
For now, I would hide the entire `File` menu. Once we have proper implementation of "Export Current Image" for Jupyter, we can add it back.